# 14th_homework

여러번의 시도 끝에 day15_final을 도출했습니다. 
한 파일 안에 다 써줘서 이게 맞는지는 모르겠지만, 파일 상에서 1개의 오류가 잡힌다고 뜨는 데에 반해
막상 작동시키면 잘 작동합니다. 
